llvm/lib/CodeGen/TargetPassConfig.cpp

```diff
@@ -1122,6 +1122,10 @@ bool TargetPassConfig::addISelPasses() {
   if (YkShadowStack) {
     addPass(createYkShadowStackPass());
   }
+
```

Comment:
Let's add a comment here to say that this is a bit of a hack, since this pass is already being run above. We just need to run it here again since we want it to run on the shadow stack, which we only insert after the original TLS pass has run.

Reply:
added 👉 8d5278769e152c549ac1e546c2a071f783f14fda